datalad/distribution/create_sibling.py

```diff
@@ -102,7 +103,7 @@ def _create_dataset_sibling(
         # TODO: Is this condition valid for != '.' only?
         path_exists = True
         try:
-            out, err = ssh(["ls", remoteds_path])
+            out, err = ssh("ls {}".format(sh_quote(remoteds_path)))
```

Comment:
Sadly not. We need to be able to prepend the PATH var adjustment to the final command. I could not figure out how to do that with a list -- short of merging the list into a string, and get into all kind of issue.